test/crs/fwdbwd.jl

```diff
@@ -1,119 +1,95 @@
-        c1 = LatLon(T(0), lon)
-        if indomain(OrthoSouth, c1)
-          c2 = convert(OrthoSouth, c1)
-          c3 = convert(LatLon, c2)
-          @test isclose(c3, c1; atol=T(0.5) * °)
-        end
-      end
-    elseif C <: TransverseMercator
-      # TODO: fix backward implementation
-      continue
-    elseif C <: Sinusoidal
-      # coordinates at the singularity of the projection (lat ≈ ±90) cannot be inverted
-      for lat in T.(-89:89), lon in T.(-180:180)
-        c1 = LatLon(lat, lon)
-        if indomain(Sinusoidal, c1)
-          c2 = convert(Sinusoidal, c1)
-          c3 = convert(LatLon, c2)
-          @test isclose(c3, c1)
-        end
-      end
-    elseif C <: LambertAzimuthal
-      # coordinates at the singularity of the projection (lat ≈ ±90) cannot be inverted
-      # Float32 inversion is not very accurate
-      if T === Float32
-        # accuracy is better at coordinates far from the edge of the projection (lon ≈ ±180)
-        for lat in T.(-89:89), lon in T.(-170:170)
-          c1 = LatLon(lat, lon)
-          if indomain(C, c1)
-            c2 = convert(C, c1)
-            c3 = convert(LatLon, c2)
-            @test isclose(c3, c1; atol=1.0f-2°)
-          end
-        end
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/40
+    PRJ <: TransverseMercator && continue
 
-        for lat in T.(-89:89), lon in T[-180:-171; 171:180]
-          c1 = LatLon(lat, lon)
-          if indomain(C, c1)
-            c2 = convert(C, c1)
-            c3 = convert(LatLon, c2)
-            @test isclose(c3, c1; atol=1.0f0°)
-          end
-        end
-      else
-        for lat in T.(-89:89), lon in T.(-180:180)
-          c1 = LatLon(lat, lon)
-          if indomain(C, c1)
-            c2 = convert(C, c1)
-            c3 = convert(LatLon, c2)
-            @test isclose(c3, c1; atol=1e-6°)
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/55
+    PRJ <: Robinson && continue
+
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/265
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/268
+    PRJ <: LambertAzimuthal && continue
+
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/272
+    PRJ <: OrthoNorth && continue
+    PRJ <: OrthoSouth && continue
+
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/295
+    PRJ <: Sinusoidal && continue
+
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/296
+    PRJ <: Albers && continue
+
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/297
+    PRJ <: GallPeters && continue
+
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/298
+    PRJ <: Behrmann && continue
+
+    # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/299
+    PRJ <: LambertCylindrical && continue
+
+    # loop over all possible latitude and longitude values
+    # that should be recovered by the given PRJ type
+    success = true
+    for (lat, lon) in Iterators.product(T.(-90:90), T.(-180:180))
```

Comment:
Currently, we loop over all possible values of latitude and longitude, and then check if the pair is `indomain` before the round-trip test. Perhaps we need a new API function `isrecoverable(PRJ, ll)` that also checks when a value in the domain is recoverable, i.e., satisfies the round-trip conversion.

Alternatively, we could introduce a test utility function `recoverablelatlon(PRJ)` that only returns the values of latitude and longitude that are expected to survive the round-trip conversion.